sequencer/src/bin/deploy.rs

```diff
@@ -111,6 +112,20 @@ struct Options {
     #[clap(long, env = "ESPRESSO_SEQUENCER_PERMISSIONED_PROVER")]
     permissioned_prover: Option<Address>,
 
+    /// A toml file with the initial stake table.
+    ///
+    /// Schema:
+    ///
+    /// public_keys = [
+    ///   {
+    ///     stake_table_key = "BLS_VER_KEY~...",
+    ///     state_ver_key = "SCHNORR_VER_KEY~...",
+    ///     da = true,
```

Comment:
Good point. I think in the sequencer we shouldn't really use PeerConfigKeys but have our own type (and also not be generic over the pubkey). However for loading the initial stake table I think it's fine to use PeerConfigKeys because this is the type that I think the orchestrator loads from file on startup and initially we will put the same stake table that was used by the orchestrator into the contract. However for this version we don't need a stake amount field so I chose not to add it. 

We can add it to the initial stake table file or somehow make the rust code work without it. I think both are fine.